SimpleClassicThemeTaskbar/Helpers/Logger.cs

```diff
@@ -41,12 +46,17 @@ public static void Initialize(LoggerVerbosity verbosity)
             if (loggerOff)
                 return;
 
-            fs = new FileStream("latest.log", FileMode.Create, FileAccess.Write, FileShare.Read);
+            fs = new FileStream(FilePath = "latest.log", FileMode.Create, FileAccess.Write, FileShare.Read);
+            sw = new StreamWriter(fs) { AutoFlush = true };
             Log(LoggerVerbosity.Basic, "Logger", "Succesfully initialized logger");
 
             Log(LoggerVerbosity.Detailed, "SystemDump", "Performing quick system dump");
             Log(LoggerVerbosity.Detailed, "SystemDump", $"OS: {RuntimeInformation.OSDescription} {RuntimeInformation.OSArchitecture}");
-            if (ApplicationEntryPoint.SCTCompatMode) Log(LoggerVerbosity.Detailed, "SystemDump", $"SCT version: {Assembly.LoadFrom("C:\\SCT\\SCT.exe").GetName().Version}");
+
+            // HACK: This uses a fixed path, this needs to be replaced
```

Comment:
Don't think we need to replace this. When SCTCompatMode == true it's safe to assume that SCT will always be located in C:\SCT\SCT.exe.

Reply:
Fair, this should be addressed in SCT generally though. A fixed installation path is not good.

Reply:
Maybe when we have libsct ready this will be easier to manage? For now I don't see an easy way to resolve it.

Reply:
I'll remove the comment for now, but we'll have to keep track of this.